### ISSUE TITLE
Improve accessibility of Buzzer simulator

### ISSUE THREAD

**Issue by jamesadevine** [microsoft/jacdac#527]
**Describe the bug**

User feedback:

> Buzzer. This was not specifically an access barrier but I was not sure of the purpose of this. After the buzzer, I hear some characters. As I navigate through the page, I hear “A W S E D F T G Y H U J K O L P ; '

**Comment by pelikhan** [comment]
Hidden piano from aria as it is not accessible.
